spec/cell_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+RSpec.describe Cell do
+  before(:each) do
+    @cell = Cell.new("B4")
+    @cruiser = Ship.new("Cruiser", 3)
+    @cell_1 = Cell.new("B4")
+  end
+
+  describe '#initialize' do
+    it 'exists' do
+      expect(@cell).to be_a(Cell)
+    end
+
+    it 'has attributes' do
+      expect(@cell.coordinate).to eq("B4")
+    end
+  end
+
+  describe '#ship' do
+    it 'starts with ship being nil' do
+      expect(@cell.ship).to be(nil)
+    end
+  end
+
+  describe '#empty' do
+    it 'has a default of empty' do
+      expect(@cell.empty?).to be(true)
+    end
+
+    it 'returns true if a ship is placed' do
+      @cell.place_ship(@cruiser)
+
+      expect(@cell.empty?).to be(false)
+    end
+  end
+
+  describe '#place ship' do
+    it 'can add a ship to a cell' do
+      @cell.place_ship(@cruiser)
+
+      expect(@cell.ship).to eq(@cruiser)
+      expect(@cell.ship).to be_a(Ship)
+    end
+  end
+
+  describe '#damaged' do
```

Comment:
hey @sarahgarlock ! I totally understand why you put #damaged in ln 47 based on interaction pattern description that says "When it is fired upon, the cell’s ship should be damaged if it has been fired upon". But i think we should change it to fired upon since that's the name of the method we're calling and I don't see a damaged method in the upcoming interaction pattern.

Reply:
Changed #damaged to #fired upon